x-pack/platform/plugins/shared/fleet/common/types/rest_spec/agentless_policy.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type TypeOf, schema } from '@kbn/config-schema';
+
+import {
+  PackagePolicyResponseSchema,
+  SimplifiedCreatePackagePolicyRequestBodySchema,
+} from '../models/package_policy_schema';
+
+export const CreateAgentlessPolicyRequestSchema = {
+  query: schema.object({
+    format: schema.oneOf([schema.literal('legacy'), schema.literal('simplified')], {
+      defaultValue: 'simplified',
+      meta: {
+        description: 'The format of the response package policy.',
+      },
+    }),
+  }),
+  body: SimplifiedCreatePackagePolicyRequestBodySchema.extends({
+    cloud_connector_id: schema.maybe(
+      schema.nullable(
+        schema.string({
+          meta: {
+            description: 'ID of the cloud connector associated with this package policy.',
+          },
+        })
+      )
+    ),
+    supports_cloud_connector: schema.maybe(
```

Comment:
What would be the roll of the `supports_cloud_connector` field? Is the presence of a `cloud_connector_id` enough to suggest as much or is there an intermediate state where an agentless policy can support cloud connectors, but not have a relationship yet?